### ISSUE TITLE
OP_REQUIRES failed at save_restore_v2_ops.cc:184

### ISSUE THREAD

**Issue by hlitz** [haotianteng/Chiron#80]
**Describe the bug**
script fails with:
...
INFO:tensorflow:Restoring parameters from chiron/model/DNA_default/model.ckpt-209071
INFO:tensorflow:Restoring parameters from chiron/model/DNA_default/model.ckpt-209071
2019-02-12 20:07:14.446087: W tensorflow/core/framework/op_kernel.cc:1273] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key res_layer1/branch1/conv1_bn/offset not \
found in checkpoint

**To Reproduce**
python chiron/entry.py call -i chiron/example_data/DNA/ -m chiron/model/DNA_default/ -o out/

**Environment (please complete the following information):**
Ubuntu
2xGTX2080ti
chiron: master branch: f21d7bea5d0c1c8ac071181bd777301df69e8b7d
conda with recent packages

**Additional context**

**Comment by haotianteng** [comment]
Please use the Chiron release version. As I am testing different batch normalization option.
Use Chiron-v0.4, which has exactly the same DNA model as the current branch.

**Comment by hlitz** [comment]
Could you create a branch for each release version you put out? I need source code access and it would be great to always have a stable branch to work with. What branch/commit would you suggest I should work with?

**Comment by haotianteng** [comment]
Check it out:
https://github.com/haotianteng/Chiron/releases
